packages/edit-post/src/components/header/writing-menu/index.js

```diff
@@ -19,17 +19,32 @@ function WritingMenu( { onClose } ) {
 				feature="fixedToolbar"
 				label={ __( 'Top Toolbar' ) }
 				info={ __( 'Access all block and document tools in a single place' ) }
-				onToggle={ onClose } />
+				onToggle={ onClose }
+				messages={ {
```

Comment:
My concern here is that we pass an object constructor as prop. This will force `FeatureToggle` to re-render whenever `WritingMenu` re-renders which isn't optimal. Can we pass two string props instead to avoid such case?

Reply:
I guess we can. Just noting there are many, many, cases in the codebase where this pattern is used. If it's a real concern, then maybe it should be a guideline.

Reply:
I think there is a post which @youknowriad wants to write about performance and how to optimize towards it. I hope we can use it as a template for guidelines later.